package.json

```diff
@@ -57,6 +57,7 @@
     "eslint-plugin-mocha": "^4.11.0",
     "eslint-plugin-react": "^7.11.0",
     "husky": "^1.1.0",
+    "lolex": "^3.0.0",
```

Comment:
?

Reply:
i'm using it in order to manipulate the test's clock.